linera-chain/src/chain.rs

```diff
@@ -184,7 +184,7 @@ where
                 event.is_none(),
                 ChainError::MissingCrossChainUpdate {
                     chain_id,
-                    origin,
+                    origin: origin.into(),
```

Comment:
Isn't `*origin` shorter / more precise?

Reply:
No, this one is different from the one in `linera-core/src/node.rs`. This converts from `Origin` to `Box<Origin>`, while the other from `Box<Origin>` to `Origin`.